packages/visx-demo/src/sandboxes/visx-shape-splitlinepath/generateSnakePath.ts

```diff
@@ -0,0 +1,58 @@
+interface Point {
+  x: number;
+  y: number;
+}
+
+const pointKey = ({ x, y }: Point) => [x, y].join('_');
+
+function distance(a: Point, b: Point) {
+  return Math.sqrt((b.x - a.x) ** 2 + (b.y - a.y) ** 2);
+}
+
+export default function generateSnakePath({
```

Comment:
generate a continuous path that fill rectangular space, similar to the old Nokia snake game.

Reply:
could we commit that? 😆 

```suggestion
// generate a continuous path that fill rectangular space, similar to the classic Nokia snake game.
export default function generateSnakePath({
```